iib/workers/tasks/build_create_empty_index.py

```diff
@@ -112,7 +112,7 @@ def handle_create_empty_index_request(
                 temp_dir=temp_dir,
                 from_index_resolved=from_index_resolved,
                 from_index=from_index,
-                binary_image=binary_image,
+                binary_image=prebuild_info['binary_image'],
```

Comment:
I am sorry I do not follow... could you explain? 

I see it this way:
`binary_image` that was used was taken from function parameter and for my test it was not set (it was None). 
But as you can see if it is taken from `prebuild_info['binary_image']` and this value is the result of mapping from None inside of `prepare_request_for_build` by the `binary_image()` function that you might be referencing to.

So I think it was not correct since it was not working (for me).

Reply:
Tests for that is run curl without `binary_image set.